easybuild/easyconfigs/a/ANGEL/ANGEL-3.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,52 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	Custom
+#
+# Notes::
+##
+
+easyblock = 'PythonPackage'
+
+name = 'ANGEL'
+version = '3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/PacificBiosciences/ANGEL'
+description = """ANGEL: Robust Open Reading Frame prediction"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/PacificBiosciences/ANGEL/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a0319553055d3dfc84a4f732ed246c180c23ee9c397810c96acd7940721ae57d']
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('Biopython', '1.73'),
+    ('scikit-learn', '0.20.3'),
+]
+
+download_dep_fail = True
+
+use_pip = False
+
+local_bin_files = [
+    'dumb_predict.py',
+    'angel_make_training_set.py',
+    'angel_train.py',
+    'angel_predict.py',
+    'angel_compare_files.py'
+]
+
+options = {'modulename': False}
+
```

Comment:
@robqiao please add here:
```
sanity_pip_check = True
```

(@boegel this is not checked by the tests?)

Reply:
@robqiao any update on the request from @smoors ?

Reply:
https://github.com/DeepThoughtHPC/easybuild-easyconfigs/pull/8

Reply:
@robqiao Can you be a bit more specific? Which dep versions will `pip` overwrite when enabling `use_pip`?